kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/QueryDataServiceIntegrationTest.java

```diff
@@ -42,19 +43,22 @@
 import org.kie.server.api.util.QueryFilterSpecBuilder;
 import org.kie.server.client.KieServicesClient;
 import org.kie.server.client.QueryServicesClient;
+import org.kie.server.integrationtests.category.UnstableOnJenkinsPrBuilder;
 import org.kie.server.integrationtests.config.TestConfig;
 import org.kie.server.integrationtests.shared.KieServerDeployer;
 
 import static org.junit.Assert.*;
 import static org.junit.Assume.assumeFalse;
 
+// Skip for PR builder as creating of container for query-definition-project on Kie server side takes extreme amount of time (10 minutes+)
+@Category({UnstableOnJenkinsPrBuilder.class})
```

Comment:
I disagree with this as we can go ahead and just disable tests for kie server completely because of inefficient CI environment